### ISSUE TITLE
Issues with optional Precision mod

### ISSUE THREAD

**Issue by Gravecat** [Lost-Outpost/lost-legacy#93]
When using the optional Precision mod (and the Precision - Nemesis Output, with LostLegacy_Nemesis Output disabled), I've noticed two bugs that occur:

Firstly, the "The Ultimate Dodge Mod" MCM menu never shows up in the list, so it's impossible to sneak or rebind the sneak key.

Secondly, when playing as a Lilmothiit (I haven't tested on other races), becoming a vampire causes the character model to become completely deformed and broken.

Neither of these bugs appear to occur when Precision and Precision - Nemesis Output are disabled.

**Comment by GreatPadinski** [comment]
Yeah, the optional mods may cause some funkiness as warned in the Configuration guide https://github.com/Lost-Outpost/lost-legacy/blob/main/CONFIGURATION.md#optional-content
